server/public/model/link_metadata.go

```diff
@@ -89,7 +89,7 @@ func (o *LinkMetadata) PreSave() {
 }
 
 func (o *LinkMetadata) IsValid() *AppError {
-	if o.URL == "" {
+	if o.URL == "" || len(o.URL) > 2048 {
```

Comment:
Can you please add this number to `const` declaration above? Such as `LinkMetadataMaxURLLength`? I'd also rather have a test case for this if you have bandwidth to add, please feel free to do so. If not, let us know and we can help with that.

Reply:
I will add the const, but I don't think I'll have time to write a test.

I would appreciate your help :)

Reply:
I will add the const, but I don't think I'll have time to write a test.

I would appreciate your help :)

Reply:
I have added the requested constant and also had a minute to add a testcase.